src/reanimated2/animation/springUtils.ts

```diff
@@ -52,6 +58,38 @@ export interface InnerSpringAnimation
   toValue: number;
   current: number;
 }
+export function validateConfig(config: DefaultSpringConfig): boolean {
+  'worklet';
+  let errorLog = '[Reanimated] Invalid spring config, ';
+  let configIsInvalid = false;
+
+  (
+    [
+      'stiffness',
+      'damping',
+      'dampingRatio',
+      'restDisplacementThreshold',
+      'restSpeedThreshold',
+      'mass',
+    ] as const
+  ).forEach((property) => {
```

Comment:
In React world, `prop` is usually used here.
```suggestion
  ).forEach((prop) => {
```